tests/acceptance/visual-regression-test.js

```diff
@@ -18,26 +18,27 @@ module('Acceptance | visual regression', function(hooks) {
       { route: '/legal/', title: '/legal.html' },
       { route: '/', title: '/index.html' },
       { route: '/mascots/', title: '/mascots.html' },
-      { route: '/meetup-assets/', title: '/meetup-assets.html' },
```

Comment:
this should be added back in as `{ route: '/meetups/assets/', title: '/meetup-assets.html' },`

Reply:
or `/community/meetups/assets` or whatever the right url is 🙈

Reply:
I'm adding it as `/community/meetups/assets, title: '/community/meetups/assets.html'`
does that look right?

Reply:
no it needs to be `/community/meetups/assets, title: '/meetup-assets.html'`

the "title" refers to the url that it was on the **old** website. It is used for us to compare the new and the old website on Percy 👍